explore/src/main/scala/explore/observationtree/ObsListActions.scala

```diff
@@ -72,10 +72,8 @@ object ObsListActions {
         elemWithIndexOpt.fold {
           ProgramDeleteObservation.execute[IO](obsId).void
         } { case (obs, _) =>
-          ProgramCreateObservation
-            .execute[IO](CreateObservationInput(programId = "p-2", observationId = obs.id.assign))
-            .void >>
-            focusedObs.set(obs.id.some).to[IO]
+          // Not much to do here, the observation must be created before we get here
```

Comment:
It's not straight forward because we need the result of the insertion. Even if we committed to a longer wait for the server round trip, if we don't have the new id, we won't be able to easily select it. We could look for a "new observation", but that seems a bit sketchy to me. We do have the possibility of other users. Plus, that'd be a lot more state to look after.